src/app.py

```diff
@@ -1,5 +1,15 @@
+from flask import Flask
 import connexion
+from flask_pymongo import pymongo
+import repositories.db as DB
```

Comment:
`flask_pymongo` is not found in `requirements.txt` which means you might have installed globally on your local dev. Pls fix this